Makefile

```diff
@@ -495,4 +495,4 @@ operator-bundle-update:
 	$(CONTAINER_COMMAND) push $(BUNDLE_IMAGE)
 
 operator-index-build:
-	opm index add --bundles $(BUNDLE_IMAGE) --tag $(INDEX_IMAGE) --container-tool $(CONTAINER_COMMAND)
+	opm index add --bundles $(BUNDLE_IMAGE) --tag $(INDEX_IMAGE) --container-tool podman
```

Comment:
Were we just not configuring this properly before?
I guess it feels like this could be fixed from wherever the job was being run since it's already configurable here.

Or are we not interested in keeping `docker` around at all?

Reply:
@michaellevy101 Can you explain the original reasoning? From what I know we are just moving off of docker right?

`$CONTAINER_COMMAND` is being set in a number of places. In Jenkins it is set 

1) In the Jenkinsfile: https://github.com/openshift/assisted-service/blob/master/Jenkinsfile#L7
2) In the Makefile: https://github.com/openshift/assisted-service/blob/master/Makefile#L5
3) In this hack script: https://github.com/openshift/assisted-service/blob/master/hack/utils.sh#L8-L16

Unclear how it resolves to `podman-remote` is pretty unclear to me I guess.

Reply:
So then is the answer to set `CONTAINER_COMMAND` to `podman` in those places as well, rather that hardcoding?